config/path

```diff
@@ -43,11 +43,11 @@ STAMPS=$BUILD/.stamps
 STAMPS_INSTALL=$BUILD/image/.stamps
 DOCS=DOCS
 TOOLCHAIN=$BUILD/toolchain
-SYSROOT_PREFIX=$ROOT/$TOOLCHAIN/$TARGET_NAME/sysroot
+SYSROOT_PREFIX=$TOOLCHAIN/$TARGET_NAME/sysroot
 LIB_PREFIX=$SYSROOT_PREFIX/usr
-TARGET_PREFIX=$ROOT/$TOOLCHAIN/bin/$TARGET_NAME-
+TARGET_PREFIX=$TOOLCHAIN/bin/$TARGET_NAME-
 
-FAKEROOT_SCRIPT=$ROOT/.fakeroot.$BUILD
+FAKEROOT_SCRIPT=$ROOT/.fakeroot.$(basename $BUILD)
```

Comment:
Yes, good point. 

Unfortunately I've somehow managed to totally screw up my local repo - will close this and push again.